modules/core/04-channel/keeper/keeper_test.go

```diff
@@ -860,3 +901,12 @@ func (suite *KeeperTestSuite) sendMockPackets(path *ibctesting.Path, numPackets
 		suite.Require().NoError(err)
 	}
 }
+
+// sendMockPacket sends a packet from source to dest but face the timeout.
+func (suite *KeeperTestSuite) sendMockPacketsTimeout(path *ibctesting.Path, numPackets int) {
+	for i := 0; i < numPackets; i++ {
+
+		_, err := path.EndpointB.SendPacket(clienttypes.NewHeight(1, 1000), 1, ibctesting.MockPacketData)
+		suite.Require().Error(err)
```

Comment:
I am going to push an update merging this function and `sendMockPackets` and also adjusting the test, because the way it is now these 2 packets are actually not sent because `SendPacket` returns an error. With the change I am going to push the packets will be sent and then timed out.

Reply:
These are the changes: https://github.com/cosmos/ibc-go/pull/5508/commits/a1e71ab96b03cdeb9acebcf2d889b27c9a956840